src/server/nbs/nbs_nuclear_playback.ts

```diff
@@ -41,7 +41,7 @@ export class NbsNUClearPlayback extends stream.Writable {
   /** Convenience method for directly streaming a file to the network. */
   static fromFile(filename: string, nuclearnetClient: NUClearNetClient) {
     const playback = NbsNUClearPlayback.of(nuclearnetClient)
-    const rawStream = fs.createReadStream(filename)
+    const rawStream = fs.createReadStream(filename, { highWaterMark: 1024 * 1024 * 32 })
```

Comment:
Add a comment to explain magic number?

Reply:
Unfortunately, this magic number is literally magic. I don't know or remember where it came from :P